blog-cse/2022-10-13-application-update.md

```diff
@@ -0,0 +1,86 @@
+---
+title: October 13, 2022 - Application Update
+hide_table_of_contents: false
+keywords:
+  - sumo logic
+  - service release notes
+  - cloud siem
+  - cse
+tags: [cloud siem release notes, application update, announcement]
+authors:
+  - name: Peter Kazmir
+    title: Principal Product Manager, Security Applications
+    url: https://github.com/pkazmir-sumo
+---
+
+### October 13, 2022 - Application Update
```

Comment:
I'd remove this since that's the title and RSS feed real estate is limited.

```suggestion

```

Reply:
Oh! Makes sense. Thanks!